test/integration/rails_gems_full_stack_test.rb

```diff
@@ -11,10 +11,11 @@ def setup
     # The normal relative directory lookup of coverband won't work for our dummy rails project
     Coverband.configure("./test/rails#{Rails::VERSION::MAJOR}_dummy/config/coverband.rb")
     Coverband.configuration.track_gems = true
-    Coverband.configuration.gem_details = false
+    Coverband.configuration.gem_details = true
```

Comment:
I'm not sure how this test was working with `gem_details = false` as it seems to be looking for coverage on a specific file within the gem.

Reply:
hmmm that is weird.

Reply:
My guess is that it was picking up coverage from a different test?

Reply:
hmm yeah cause without gem details it should render the gem name but not the individual files... makes me think I am likely still adding too much unneeded HTML to the report when gem defails is false.